samples: net: add graceful bring down of network interfaces (#13600)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

      The documentation is now found in the :ref:`networking_samples` section.
    * The sample to use the :ref:`coap_client_interface` library.

* :ref:`https_client` sample:

>   * Added graceful bring down of the network interfaces.

Reviewer: same applies for line no: 454.
```suggestion
  * Updated the sample to gracefully bring down the network interfaces.
```
